### PR TITLE
Event/PyCon india

### PR DESCRIPTION
**Pull request by gcdeshpande**

### CODE REVIEW

File: events/PyConIndia-2021-09-20.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Mentored Sprints at PyCon India 2021
+excerpt: >-
+  Text to be displayed in the preview. Make it catchy and descriptive
+# note this is the date for the event not when the post is published
+date: '2021-09-20'
+# add images to images/events and update the path here, you can use the same image for the thumb and the main image on the post
+thumb_image: images/PyConIndia.jpg
+image: images/PyConIndia.jpg
```
Comment (trallard) [inline events/PyConIndia-2021-09-20.md:9]: Seems like you forgot to add the images to the repo

**Review by trallard — COMMENTED** [review]
Same as #18  once resolved we can merge

File: events/PyConIndia-2021-09-20.md (hunk 1 above)
Comment (gcdeshpande) [reply events/PyConIndia-2021-09-20.md:9]: @trallard Sorry. Added the images

**Comment by trallard** [comment]
LGMT merging now

**Pull request merged**